backend/open_webui/retrieval/vector/utils.py

```
@@ -0,0 +1,212 @@
+    def __init__(self, app=None):
+        """
+        Initialize the collection mapper.
+        
+        Args:
+            app: The FastAPI app instance with state information
+        """
+        log.info(f"CollectionMapper.__init__ called with app: {app}")
+        if app is not None:
+            log.info(f"App type: {type(app)}")
+            if hasattr(app, 'state'):
+                log.info(f"App.state attributes available")
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)